src/main/java/org/gitlab/api/http/GitlabHTTPRequestor.java

```diff
@@ -270,12 +311,55 @@ private void findNextUrl() throws MalformedURLException {
         };
     }
 
+    private void submitAttachments(HttpURLConnection connection) throws IOException {
+    	String boundary = Long.toHexString(System.currentTimeMillis()); // Just generate some unique random value.
+    	connection.setDoOutput(true);
+    	connection.setRequestProperty("Content-Type", "multipart/form-data; boundary=" + boundary);
+        String charset = "UTF-8";
```

Comment:
There seems to be some odd formatting here (perhaps because of tabs vs spaces?)

Could you resolve?